### PR TITLE
Fix(variables-scss): Export in JS barrel file #DS-1503

### PR DESCRIPTION
**Pull request by curdaj**

### CODE REVIEW

**Review by pavelklibani — APPROVED** [review]

**Comment by literat** [comment]
I think it would be better to move `theme.ts` to `theme/index.ts` if possible. Because now we can import themes from `index.ts` and some parts also from `theme.ts`.

**Review by literat — APPROVED** [review]

File: packages/design-tokens/package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "build:js": "vite build",
     "build:scss": "shx mkdir -p scss && shx cp -r src/scss/generated/* src/scss/@tokens.scss scss/",
     "clean": "rimraf esm cjs umd scss types",
-    "lint": "stylelint ./src/**/*.scss",
+    "lint": "eslint ./src/**/*.ts && stylelint ./src/**/*.scss",
```
Comment (literat) [inline packages/design-tokens/package.json:33]: `"lint": "npm-run-all --serial lint:scripts lint:styles"`

File: packages/design-tokens/.eslintrc.cjs
Comment (literat) [inline packages/design-tokens/.eslintrc.cjs:1]: ❤️

Comment (literat) [reply packages/design-tokens/.eslintrc.cjs:1]: I think this
```js
module.exports = {
  extends: ['../../.eslintrc', '@lmc-eu/eslint-config-typescript', '@lmc-eu/eslint-config-jest'],

  parserOptions: {
    ecmaVersion: 'latest',
    project: './tsconfig.eslint.json',
  },
};
```

would be suficient.

For better understanding: `@lmc-eu/eslint-config-typescript` have all neccessary configuration needed for running ESLint with TypeScript. Like the definition of the `parser` or switching off some of the conflicting rules. You can see it in https://github.com/lmc-eu/code-quality-tools/blob/main/packages/eslint-config-typescript/index.js.

**Review by literat — APPROVED** [review]
Nice work. I have written a few suggestion for a better (and much smaller) ESLint configuration and also send a fixup with the ignore of the `desing-tokens` package from the root configuration.

Every package with its own ESLint configuration should be ignored by the root one so they are not in the collision (yeah, I know, this should be refactored).

**Comment by curdaj** [comment]
> Btw, in
> 
> ```js
> export const themes = {
>   themeLightDefault: {
>     colors: themeLightDefault.colors
>   },
>   themeLightOnBrand: {
>     colors: themeLightOnBrand.colors
>   },
> };
> ```
> 
> There should be a comma `,` at the end of the record -> `colors: themeLightOnBrand.colors,`

already fixed

File: packages/design-tokens/package.json (hunk 1 above)
Comment (curdaj) [reply packages/design-tokens/package.json:33]: ERROR: Task not found: "lint:scripts", lint:styles"

**Comment by curdaj** [comment]
> I think it would be better to move `theme.ts` to `theme/index.ts` if possible. Because now we can import themes from `index.ts` and some parts also from `theme.ts`.

I am not sure if I understand and where the problem is.

File: packages/design-tokens/.eslintrc.cjs
Comment (curdaj) [reply packages/design-tokens/.eslintrc.cjs:1]: Unfortunatelly is not. When I change it I have several warnings  "Strings must use singlequote  quotes"

Comment (curdaj) [reply packages/design-tokens/.eslintrc.cjs:1]: but is only warning

Comment (curdaj) [reply packages/design-tokens/.eslintrc.cjs:1]: resolved

**Review by literat — APPROVED** [review]

**Pull request merged**